DirectusPolicy types for typescript in SDK is incomplete

### Describe the Bug

I use Typescript to write an init script to create directus policies I need, but the typescript compiler don't compile because the DirectusPolicy type don't specify 'name' property.

The error return by compiler is : 
```
error TS2353: Object literal may only specify known properties, and 'name' does not exist in type 'Partial<{ id: string; icon: string; description: string | null; ip_access: string | null; enforce_tfa: boolean; admin_access: boolean; app_access: boolean; permissions: number[] | { id: number; role: string | ... 1 more ... | null; ... 5 more ...; fields: string[] | null; }[]; users: string[] | { ...; }[]; roles: st...'.
```

When I remove the 'name' property, the script compiles but, at runtime, directus API return an error because the 'name' property is required.

This is an extract from the directus answer when no 'name' property is specify :
```
 errors: [
    {
      message: 'Validation failed for field "name". Value is required.',
      extensions: { field: 'name', type: 'required', code: 'FAILED_VALIDATION' }
    }
  ],
```
 

I was able to correct the problem by manually edit the DirectusPolicy type in node_modules/@directus/sdk/dist/index.d.ts. By adding the line 'name: string;' inside the structure line 202 of index.d.ts.


Ps: I am a beginner in the typescript and directus world, so I hope the informations I gave are enough but if not don't hesitate to ask for more informations

### To Reproduce

Create a typescript project with @directus/sdk dependency and create an init script with the following content.

```
import { createDirectus, rest, authentication, createPolicy } from '@directus/sdk';

// Conenct as Admin to allow policy creation
const directus = createDirectus('http://0.0.0.0:8055/').with(authentication('json')).with(rest());
const login_res = await directus.login(process.env.ADMIN_EMAIL ?? '', process.env.ADMIN_PASSWORD ?? '');
if(login_res['access_token'] != '') {
  console.log("Log in as Admin");
} else {
  console.error("Can't connect as Admin");
}

// Create policy
const policy_res = await directus.request(
	createPolicy(
		{
			name: 'Watizat - Admin',
			icon: 'verified_user',
			description: "Accès non restreint à l'application et l'api pour gerer le site dans sa globalité",
			admin_access: true,
			app_access: true,
		})
);
```

Compile this typescript script with a compiler (I use npx on my side).

### Directus Version

v11.0.2

### Hosting Strategy

Self-Hosted (Docker Image)

### Database

_No response_

--- a/.changeset/lucky-files-prove.md
+++ b/.changeset/lucky-files-prove.md
@@ -1,0 +1,5 @@
+---
+'@directus/sdk': patch
+---
+
+Fix DirectusPolicy typescript declaration

--- a/contributors.yml
+++ b/contributors.yml
@@ -152,3 +152,4 @@
 - EdwardLi-coder
 - EdouardDem
 - Xchos
+- Lfillon

--- a/sdk/src/schema/policy.ts
+++ b/sdk/src/schema/policy.ts
@@ -8,6 +8,7 @@ export type DirectusPolicy<Schema> = MergeCoreCollection<
 	'directus_policies',
 	{
 		id: string; // uuid
+		name: string;
 		icon: string;
 		description: string | null;
 		ip_access: string | null;

